editor-examples.js

```diff
@@ -0,0 +1,347 @@
+/* jshint esversion: 6 */
```

Comment:
Not an exact copy. 
Anyway, I duplicated the entire script because it was the safest and quickest way to get the "examples" working without affecting the original exercises; I didn't have enough time to fully understand the entire JS code yet. It was just a quick hack, **I can always refactor the code later**.